src/screens/ContactScreen/index.js

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import {DataProvider, LayoutProvider, RecyclerListView} from 'recyclerlistview';
+import SimpleToast from 'react-native-simple-toast';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `SimpleToast` from 'react-native-simple-toast' is a new addition. Ensure that this package is properly installed and listed in your `package.json` dependencies.

```diff
+ import SimpleToast from 'react-native-simple-toast';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->